packages/loot-core/src/server/importers/ynab5.ts

```diff
@@ -178,11 +178,21 @@ async function importTransactions(
               entityIdMap.get(transaction.transfer_transaction_id) || null,
             subtransactions: subtransactions
               ? subtransactions.map(subtrans => {
+                  let payee = null;
+                  if (subtrans.transfer_account_id) {
+                    payee = payees.find(
+                      p =>
+                        p.transfer_acct ===
+                        entityIdMap.get(subtrans.transfer_account_id),
+                    ).id;
+                  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I suspect for bigger imports this operation will have poor performance. Would you mind pre-computing a hashmap for payees and then just using it here as a lookup?

i.e.
```
const payeeTransferActHashMap = new Map(payees.map((payee) => [payee.transfer_acct, payee]));

// ... and then inside the loop
payee = payeeTransferActHashMap[entityIdMap.get(subtrans.transfer_account_id)]?.id;

Reply:
Updated based on feedback :) let me know if this isnt quite what you were meaning :)